script/Ciel/Day 5/05_01_CIEL05_3D.txt

```diff
@@ -328,27 +346,37 @@ Arcueid won't even look me in the eye.
 [sha:c7f26e554665802e7c55aa08a528017f4ae0e47f]{
 -- Page 570, Offset 12333.
 -- 「絶対って、今更なに言ってんだよ!?　おまえな、先輩がそんなに嫌いなのか!?　恨みでもあるのか!?
-"'Not in a million years', what are you saying, now of all times!? Do you hate Senpai that much!? Do you have a grudge against her!?
+"What do you mean 'not in a million years'? Now's not the time for that!
+ Do you really hate Senpai that much!?
+ Do you have a grudge against her!?
 }
 [sha:b37466a4794b1a9559878ce34516be54d2844e43]{
 -- Page 570, Offset 12334.
 -- いや、あるんだな、あるんだろうよ！　でもそれとこれは関係ないだろ!?　いま殺すべきはヴローヴなんだから！」
-You do, you do have one! But this has nothing to do with that! Because what we have to do %{i}now%{/i} is kill Vlov!"
+No, you %{i}do%{/i} have one don't you!
+ But does that have anything to do with the problem at hand?!
```

Comment:
> But that has nothing do with this, has it?!

I'd prefer to keep it as a negative expression, rather than a straightforward question.